composer.json

```diff
@@ -0,0 +1,13 @@
+{
+	"name": "humanmade/whitelist-html",
+	"description": "Whitelist HTML for WordPress. Adds esc_*()-like function to allow some HTML.",
+	"type": "wordpress-plugin",
```

Comment:
This package as not a typical plugin for wordpress so you can remove this line - the plugin file itself doesn't have a plugin header comment so WP won't pick it up in the usual way.